modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export AIRGAP_REG='<bastion_host_name>:5000' <2>
+$ export AIRGAP_REPO='<repo_name>' <3>
```

Comment:
Suggest dropping `AIRGAP_`, because the fact that `oc` can reach both Red Hat and the mirror means the mirror itself is probably not airgapped.  [Earlier][1] I floated "local registry" or similar, so maybe `LOCAL_REGISTRY` and `LOCAL_REPO` if you wanted to follow that pattern?

[1]: https://github.com/openshift/openshift-docs/pull/16678#discussion_r325831243